### ISSUE TITLE
missing symbol called

### ISSUE THREAD

**Issue by kholiavko-roman** [ringcentral/ringcentral-softphone-ts#2]
When attempting to migrate to a new version using Opus for audio, we encounter a `missing symbol called` error on macOS M1.

![image](https://github.com/user-attachments/assets/3fb916df-4272-4dc9-8201-d51174840f1e)

`@discordjs/opus` does not seem to work with Apple silicon. The same issue has been reported to the repository https://github.com/discordjs/opus/issues/165 but since the project has no active maintainers, it was not fixed.

Also, `@discordjs/opus` has a high severity vulnerability [https://github.com/advisories/GHSA-43wq-xrcm-3vgr](https://github.com/advisories/GHSA-43wq-xrcm-3vgr)

In our project, as a workaround, we tried to override the `@discordjs/opus` dependency to https://www.npmjs.com/package/mediaplex, and it looks okay so far. 

Any chances to migrate to another library instead of `@discordjs/opus` to solve this problems?

**Comment by kholiavko-roman** [comment]
@tylerlong could you please take a look?

**Comment by twlite** [comment]
mediaplex is a drop-in replacement for discordjs/opus so this should be as easy as changing the import statement

**Comment by kholiavko-roman** [comment]
Yeah, it should be, but it could be some minor nuances that only the author knows about

**Comment by twlite** [comment]
I have left a pr, rest is up to the author https://github.com/ringcentral/ringcentral-softphone-ts/pull/5

**Comment by tylerlong** [comment]
I am aware of this issue now: "@discordjs/opus does not seem to work with Apple silicon."

Let me see what I can do.

**Comment by kholiavko-roman** [comment]
Thank

**Comment by kholiavko-roman** [comment]
By the way, I think we tested `@discordjs/opus`, and it worked fine on a Mac with Apple silicon.

**Comment by tylerlong** [comment]
I released a new version and replaced @discordjs/opus with @evan/opus.
